### PR TITLE
Colours

### PR DESCRIPTION
**Pull request by Joezo**
### Blog title
Drawing squares of colour with code

### Blog description
A quick write up of me playing around with the canvas API a couple of years ago.

#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [ ] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

File: _posts/2016-07-18-drawing-squares-of-colour-with-code.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Drawing squares of colour with code
+categories: canvas
+author: joe_warren
+tags: html5 canvas javascript
+comments: true
+---
+
+The Canvas API has been available in Chrome for a long time now, version 4 in fact. It dawned on me one evening late last year that i'd never actually played around with it. Enter my super duper creative imagination.
```
Comment (jackdcrawford) [inline _posts/2016-07-18-drawing-squares-of-colour-with-code.md]: I'd*

File: _posts/2016-07-18-drawing-squares-of-colour-with-code.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/gist-embed/2.4/gist-embed.min.js"></script>
+<code data-gist-id="f4a9f80c1580b6c06280cbc4e0aea9ca" data-gist-line="13-16"></code>
+<small>[View the full gist](https://gist.github.com/Joezo/f4a9f80c1580b6c06280cbc4e0aea9ca)</small>
+
+Next I looped through the available space on the canvas and picked a random colour for each predefined tile size and drew it. There are a lot of magic numbers in here, they just came around from me changing them and seeing what difference they made until I settled on them. That's some of the fun of drawing things with code, the tiniest of changes can produce the most dramatic results.
+
+The code isn't pretty, it's certainly not the most efficient way of doing it either. I had fun though and felt strangely proud of my colourful squares. It felt like the first time i'd successfully printed out `$_GET` in PHP, exciting, I was discovering a whole new realm of development.
+
+The point i'm trying to make is that you should play around with a technology you havn't used before, you might end up with your own colourful squares.
```
Comment (jackdcrawford) [inline _posts/2016-07-18-drawing-squares-of-colour-with-code.md]: *I'm & *haven't

File: _posts/2016-07-18-drawing-squares-of-colour-with-code.md (hunk 1 above)
Comment (jackdcrawford) [inline _posts/2016-07-18-drawing-squares-of-colour-with-code.md]: Might be nice to link to the canvas API docs here

**Comment by jackdcrawford** [comment]
Nice 👍

**Comment by jackdcrawford** [comment]
:+1:

**Comment by carolinewoodhams** [comment]
Taking a review

**Comment by carolinewoodhams** [comment]
@Joezo Having trouble viewing the colours ("This beautiful piece of art emerged") - I'm just getting a 404 error. Any ideas?

Otherwise the rest of it looks great :)

**Pull request merged**